lib/PuppeteerSharp/Browser.cs

```diff
@@ -70,14 +70,24 @@ public string WebSocketEndpoint
             return await target.Page();
         }
 
+        /// <summary>
+        /// Returns An Array of all active targets
+        /// </summary>
+        /// <returns>An Array of all active targets</returns>
+        public IEnumerable<Target> Targets() => _targets.Values.Where(target => target.IsInitialized);
```

Comment:
would it make sense to return an array instead? not sure we want this to be lazy

Reply:
Let's do it